packages.json

```diff
@@ -22188,5 +22188,19 @@
     "description": "create db util sources from csv",
     "license": "MIT",
     "web": "https://github.com/z-kk/csv2dbsrc"
+  },
+  {
+    "name": "nptr",
+    "url": "https://15b1acb3d21ebde45e8ab271e72fb4b75ad40143:x-oauth-basic@github.com/henryas/nptr",
+    "method": "git",
+    "tags": [
+      "smart",
```

Comment:
```suggestion
```
？

Reply:
@xflywind smart pointers

Reply:
yeah I mean tags should be `smart pointers` not `smart`?